scripts/ci/run_perftest.sh

```diff
@@ -265,69 +266,63 @@ curl -XPOST -H "Authorization: Token ${TEST_TOKEN}" \
 ## Run and record tests ##
 ##########################
 
-# Generate queries to test.
-query_files=""
-for usecase in window-agg group-agg bare-agg ungrouped-agg group-window-transpose-low-card group-window-transpose-high-card iot metaquery multi-measurement; do
-  for type in $(query_types $usecase); do
-    query_fname="${TEST_FORMAT}_${usecase}_${type}"
-    $GOPATH/bin/bulk_query_gen \
-        -use-case=$usecase \
-        -query-type=$type \
-        -format=influx-${TEST_FORMAT} \
-        -timestamp-start=$(start_time $usecase) \
-        -timestamp-end=$(end_time $usecase) \
-        -queries=$queries \
-        -scale-var=$scale_var > \
-      ${DATASET_DIR}/$query_fname
-    query_files="$query_files $query_fname"
-  done
-done
-
 # Generate and ingest bulk data. Record the time spent as an ingest test if
 # specified, and run the query performance tests for each dataset.
 for usecase in iot metaquery multi-measurement; do
```

Comment:
Should be relatively straightforward to factor out a configuration driven approach, where each test has a data generation strategy, and a list of query strategies.